grabbit/src/main/groovy/com/twcable/grabbit/GrabbitConfiguration.groovy

```diff
@@ -137,11 +138,27 @@ class GrabbitConfiguration {
 
     @CompileStatic
     static class PathConfiguration {
+        private static final String COMMA = ","
+        private static final String SLASH = "/"
+        private static final String EMPTY = ""
         String path
+        Collection<String> excludePaths
         Collection<String> workflowConfigIds
+        
+        Collection<String> modifyExcludePaths(@Nonnull Collection<String> excludePaths)
+        {
```

Comment:
Minor: Doesn't follow the style conventions in the rest of the code base. If you're using IDEA and generate the project with `./gradlew idea` then it should be automatically set up to format things consistently. (Including things like tabs vs spaces.)